src/main.py

```diff
@@ -0,0 +1,44 @@
+from collection import HTTPPacketCollector
+from statistics_management import StatisticsManager
+from alerting import AlertManager
+from argparse import ArgumentParser
+from time import sleep
+import utils
+
+logger = utils.get_logger(__name__)
+
+def main(args):
+    interval_count = 0
+    statistics_check_interval_seconds = 10
+    alert_check_interval = 12
```

Comment:
For configurability, maybe make these program arguments too with these values as default? Should update the README too if so.